### PR TITLE
[ROCm][AOTI] add CK backend

### PR DESCRIPTION
**Pull request by tenpercent**
Companion to #134379 

cc @jeffdaily @sunway513 @jithunnair-amd @pruthvistony @ROCmSupport @dllehr-amd @jataylo @hongxiayang @naromero77amd @voznesenskym @penguinwu @EikanWang @jgong5 @Guobing-Chen @XiaobingSuper @zhuhaozhe @blzheng @wenzhe-nrv @jiayisunx @ipiszy @yf225 @chenyang78 @kadeng @muchulee8 @ColinPeppler @amjames @desertfire @chauhang

### CODE REVIEW

File: test/inductor/test_ck_backend.py [hunk 1]
```diff
@@ -3,6 +3,10 @@
 import os
 import unittest
 
+try:
+    from .test_aot_inductor_utils import AOTIRunnerUtil
+except ImportError:
+    from test_aot_inductor_utils import AOTIRunnerUtil
```
Comment (ColinPeppler) [inline test/inductor/test_ck_backend.py:9]: I found just this works the best :)

**Comment by ColinPeppler** [comment]
Everything LGTM other than linter. When I refactor the CUTLASS codepath, I will look into seeing what is easily re-usable here

**Review by ColinPeppler — APPROVED** [review]

**Review by chenyang78 — APPROVED** [review]
LGTM. Thanks!

**Comment by tenpercent** [comment]
Link #125453

**Pull request closed without merging**